algorithms/C/tree/HeightOfTree.c

```diff
@@ -0,0 +1,77 @@
+// Wap to find the height in a binary search tree.
+
+#include<stdio.h>
+#include<stdlib.h>
+
+struct Node{
+    int data;
+    struct Node *left;
+    struct Node *right;
+};
+
+// creating a Node
+struct Node * createNode(int data){
+	struct Node *n;
+    n = (struct Node *)malloc(sizeof(struct Node)); // allocating the memory in the heap
+    n -> data = data; // setting the data
+    n -> left = NULL; // setting left node to be NULL
+    n -> right = NULL; // setting right node to be NULL
+    return n; // returning the created node
+}
+
+// to find the max

```

Comment:
The sentence is not fully completed `to find max` please clarify it neatly

Reply:
Do i need to change it in the code and commit again?